conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+azure:
+  store_build_artifacts: true
```

Comment:
Since we are done debugging the packages, we can turn this off. Can always it back when needed

```suggestion
```